docs/policy/software-support.md

```diff
@@ -1,143 +1,149 @@
-    - BeStMan: encourage the site to transition to GridFTP, as they cannot retire GUMS until BeStMan doesn't depend on it.
-      If CMS/ATLAS, encourage them to consult their US-ATLAS/US-CMS mailing lists for assistance.
-
-1. Ask them to follow the relevant instructions for their authorization solution 
-
-    - [edg-mkgridmap](http://www.opensciencegrid.org/docs/security/lcmaps-voms-authentication/#migrating-from-edg-mkgridmap)
-    - [GUMS](http://www.opensciencegrid.org/docs/security/lcmaps-voms-authentication/#migrating-from-gums)
-
-1. After they've completed the above instructions, use one of the following methods (in order of preference) to verify 
-   LCMAPS VOMS mappings:
-
-    - If the host is a CE, verify that they are still receiving pilots:
-
-        - Query their CE directly:
-
-                :::console
-                $ condor_q -name <CE HOSTNAME> -pool <CE HOSTNAME>:9619
-
-            This may not work if the site has a strict firewall or do not run an HTCondor-CE.
+-   We are a small community and we need to take good care of our users.
+    Please be friendly and patient even when the user is frustrated or lacking in knowledge.
+-   Always sign your ticket with your full name, so people know who is responding.
+    If it's easy for you, include a signature at the bottom of your response.
+-   If you need to collect information about a problematic host, ask users to run `osg-system-profiler`.
+    It can shorten the number of times you ask for information because it collects quite a bit for you.
+-   If you run across a problem that has a chance of being hit by other users, consider:
+    -   Is there a bug we should fix in the software?
+        Or something we could improve in the software?
+    -   Is there a way to improve our documentation?
+    -   Can you extend our troubleshooting documents to help people track this down more quickly?
+        Consider the troubleshooting documents to be as much for us as for our users.
+    -   Noting it during the support discussion during the weekly OSG Technology meeting
 
-        - Add factory ops or the relevant ATLAS (T2 vs T3) support center to the ticket under `OSG Support Centers`
-        - For non-ATLAS sites, verify that the site's [pilot numbers](http://gfactory-2.opensciencegrid.org/factory/monitor/factoryStatus.html?entry=OSG_US_UConn_gluskap&frontend=OSG_Flock_frontend&infoGroup=running&elements=StatusRunning,ClientGlideRunning,ClientGlideIdle,&rra=0&wi)
-       are non-zero.
+Triage Duty
+-----------
 
-    - If the host is a GridFTP server, verify file transfer with their VO support center.
+During normal work hours, the OSG Technology Team splits responsibilities for managing incoming OSG Software support
+requests based upon a [weekly rotation](https://github.com/opensciencegrid/osg-triage-assignments/blob/master/rotation.txt).
+The OSG uses [Freshdesk](#freshdesk) to track support issues so you will need a Freshworks account with agent privileges
```

Comment:
You should put a paragraph break above this sentence.

Reply:
Alternatively, move the "During normal work hours..." sentence to just above "If you are on triage duty..."